js/base-views.js

```diff
@@ -0,0 +1,268 @@
+								<span class="slider round"></span>
+							</label></div>
+						</label>
+						<div class="pagedetails__header w-100">
+							<h3 class="page_title text-capitalize">Default view</h3>
+						</div>
+						<label class="split wth__row">
+							<span>Custom name</span>
+							<input name="views0Name" placeholder="Default">
+						</label>
+						<div class="pagedetails__header w-100">
+							<h3 class="page_title text-capitalize">View 1</h3>
+						</div>
+						<div class="split">
+							<label class="half">
+								<span>Enable view 1</span>
+								<input type="checkbox" name="views1Enable">
+							</label>
+							<label class="half">
+								<span class="help" title="${mutExclusiveHelp}">Mutually exclusive with previous</span>
+								<input type="checkbox" name="views1Exclusive">
+							</label>
+						</div>
+						<label class="split wth__row">
+							<span>Custom name</span>
+							<input name="views1Name" placeholder="View 1">
+						</label>
+						<div class="pagedetails__header w-100">
+							<h3 class="page_title text-capitalize">View 2</h3>
+						</div>
+						<div class="split">
+							<label class="half">
+								<span>Enable view 2</span>
+								<input type="checkbox" name="views2Enable">
+							</label>
+							<label class="half">
+								<span class="help" title="${mutExclusiveHelp}">Mutually exclusive with previous</span>
+								<input type="checkbox" name="views2Exclusive">
+							</label>
+						</div>
+						<label class="split wth__row">
+							<span>Custom name</span>
+							<input name="views2Name" placeholder="View 2">
+						</label>
+						<div class="pagedetails__header w-100">
+							<h3 class="page_title text-capitalize">View 3</h3>
+						</div>
+						<div class="split">
+							<label class="half">
+								<span>Enable view 3</span>
+								<input type="checkbox" name="views3Enable">
+							</label>
+							<label class="half">
+								<span class="help" title="${mutExclusiveHelp}">Mutually exclusive with previous</span>
+								<input type="checkbox" name="views3Exclusive">
+							</label>
+						</div>
+						<label class="split wth__row">
+							<span>Custom name</span>
+							<input name="views3Name" placeholder="View 3">
+						</label>
+					</div>
+				`).appendTo($("body"));
+
+				const handleProp = (propName) => $dialog.find(`[name="${propName}"]`).each((i, e) => {
+					const $e = $(e);
+					if ($e.is(":checkbox")) {
+						$e.prop("checked", !!page.get(`bR20cfg_${propName}`));
+					} else {
+						$e.val(page.get(`bR20cfg_${propName}`));
+					}
+				});
+				const props = [
+					"viewsEnable",
+					"views0Name",
+					"views1Enable",
+					"views1Exclusive",
+					"views1Name",
+					"views2Enable",
+					"views2Exclusive",
+					"views2Name",
+					"views3Enable",
+					"views3Exclusive",
+					"views3Name",
+				];
+				props.forEach(handleProp);
+
+				function doSaveValues () {
```

Comment:
prefer arrow functions `const doSaveValues = () => { ... }` to avoid clobbering `this` (and save a few characters!)